core/drivers/firewall/stm32_etzpc.c

```diff
@@ -153,6 +156,43 @@ static enum etzpc_decprot_attributes etzpc_binding2decprot(uint32_t mode)
 	}
 }
 
+static void
+sanitize_decprot_config(uint32_t decprot_id __maybe_unused,
+			enum etzpc_decprot_attributes attr __maybe_unused)
+{
+#ifdef CFG_STM32MP15
+	/*
+	 * stm32MP15: check dependency on RCC TZEN/MCKPROT configuration
+	 * when a ETZPC resource is assignted to secure or isolated Cortex-M
+	 * coprocessor.
+	 */
+	switch (attr) {
+	case DECPROT_S_RW:
+	case DECPROT_NS_R_S_W:
+		if (!stm32_rcc_is_secure()) {
+			IMSG("WARNING: RCC not secure, insecure ETZPC hardening %"PRIu32":%u",
+			     decprot_id, attr);
+			if (!IS_ENABLED(CFG_INSECURE))
+				panic();
+		}
+		break;
+	case DECPROT_MCU_ISOLATION:
+		if (!stm32_rcc_is_secure() || !stm32_rcc_is_mckprot()) {
```

Comment:
This trace does not inform if rcc isn't secure

Reply:
I think tt does: MCU isolation requires`stm32_rcc_is_secure() == true && stm32_rcc_is_mckprot() == true`